azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/features/SettingsFeatures.kt

```diff
@@ -47,6 +48,19 @@ class SettingsFeatures {
                 .getSharedPreferences(FEATURE_FLAG_SHARED_PREFS_KEY, Context.MODE_PRIVATE)
                 .edit().putBoolean(IS_LANGUAGE_SETTING_ENABLED, isFeatureEnabled).apply()
         }
+
+        fun getIsCustomTranslationEnabled(context: Context): Boolean {
+            return context.applicationContext
+                .getSharedPreferences(FEATURE_FLAG_SHARED_PREFS_KEY, Context.MODE_PRIVATE)
+                .getBoolean(LANGUAGE_CUSTOM_TRANSLATION_ENABLE, false)
+        }
+
+        fun getCustomTranslationMap(): Map<String, String> {
```

Comment:
Use kotlin mapOf syntax

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/map-of.html